python-sdk/tests/benchmark/analyse.py

```diff
@@ -131,24 +167,55 @@ def analyse_results(results_filepath):
 
     summary = mean_by_dag[SUMMARY_FIELDS]
 
+    content = ""
     # print Markdown tables per database
     for database_name in summary["database"].unique().tolist():
-        print(f"\n### Database: {database_name}\n")
-        print(summary[summary["database"] == database_name].to_markdown(index=False))
+        content = content + f"\n\n### Database: {database_name}\n"
+        content = content + str(
+            summary[summary["database"] == database_name].to_markdown(index=False)
+        )
```

Comment:
I think it is worth refactoring this into something like a `get_content` method.

Reply:
@feluelle Thanks, I have added it :)